src/api/java/com/minecolonies/api/entity/mobs/util/MobEventsUtils.java

```diff
@@ -90,6 +92,23 @@ public static void raiderEvent(final World world, final IColony colony)
         BarbarianEventUtils.barbarianEvent(world, colony, targetSpawnPoint, raidNumber, horde);
     }
 
+    /**
+     * Checks if a raid is triggered
+     */
+    public static boolean checkForRaid(final IColony colony)
+    {
+        if (colony.getWorld().getDifficulty() != EnumDifficulty.PEACEFUL
+              && Configurations.gameplay.doBarbariansSpawn
+              && colony.getRaiderManager().canHaveRaiderEvents()
+              && !colony.getPackageManager().getOfficerSubscribers().isEmpty()
+              && MobEventsUtils.isItTimeToRaid(colony.getWorld(), colony))
+        {
+            MobEventsUtils.raiderEvent(colony.getWorld(), colony);
+        }
+
+        return false;
```

Comment:
Currently this always returns false, might want to change that =P

Reply:
oh the false is there because it previously was used as a transition in the colonys statemachine, but it isn't anymore, so ye would make sense to change it to reflect whether a raid is triggered or not :)